packages/frontend/src/components/NavBar/AiAgentsButton.tsx

```diff
@@ -68,5 +40,17 @@ export const AiAgentsButton = () => {
         return null;
     }
 
-    return <Button projectUuid={projectUuid} />;
+    return (
+        <Button
+            size="xs"
+            variant="default"
+            fz="sm"
+            leftIcon={
+                <MantineIcon icon={IconMessageCircleStar} color="#adb5bd" />
+            }
+            onClick={() => navigate(`/projects/${projectUuid}/ai-agents`)}
```

Comment:
can this be a `<Button component={Link} to="/blabla">` where Link is
`import { Link } from 'react-router'` ?